### PR TITLE
fixing cppcheck oneMKL/oneTBB download scripts and launcher scripts for hadoop/spark/scala

### PR DESCRIPTION
**Pull request by napetrov**
# Description
fixing cppcheck oneMKL and oneTBB download scripts

### CODE REVIEW

File: dev/download_tbb.sh [hunk 1]
```diff
@@ -49,14 +48,14 @@ if [ ! -d "${DST}/${OS}/bin" ]; then
...
 
-  echo tar -xvf "${DST}/${TBB_PACKAGE}.tgz" -C ${DST}
-  tar -C ${DST}/${OS} --strip-components=1 -xvf "${DST}/${TBB_PACKAGE}.tgz" ${TBB_VERSION}
-  echo "Downloaded and unpacked Intel(R) TBB to ${DST}/${OS}"
+  echo tar -xvf "${DST}/${TBB_PACKAGE}.tgz" -C "${DST}"
+  tar -C "${DST}/${OS}" --strip-components=1 -xvf "${DST}/${TBB_PACKAGE}.tgz" ${TBB_VERSION}
+  echo "Downloaded and unpacked oneTBB to ${DST}/${OS}"
 else
-  echo "Intel(R) TBB is already installed in ${DST}/${OS}"
+  echo "IoneTBB is already installed in ${DST}/${OS}"
```
Comment (vepifanov) [inline dev/download_tbb.sh:60]: Should it be just "oneTBB"?

Comment (napetrov) [reply dev/download_tbb.sh]: Thanks, fixed

**Pull request merged**